test/e2e/antreapolicy_test.go

```diff
@@ -284,6 +284,51 @@ func testInvalidACNPNoPriority(t *testing.T) {
 	}
 }
 
+func testInvalidANPIngressPeerGroupSetWithPodSelector(t *testing.T) {
```

Comment:
Is there a case for adding tests where ANPs/ACNPs are created with a group selector, or updated by adding/removing a group selector?